client/packages/host/package.json

```diff
@@ -37,7 +37,6 @@
     "history": "^5.1.0",
     "react": "^18.0.0",
     "react-dom": "^18.0.0",
-    "react-router-dom": "^6.0.2",
```

Comment:
this shouldn't be here! it's imported by common ( and a different version, at that ) and can be imported from `@common` if required